tests/DisplayProjectsTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once 'src/Services/ProjectDisplayService.php';
+require_once 'src/Models/ProjectModel.php';
+
+class DisplayProjectsTest extends TestCase
```

Comment:
So both of the tests you've written are testing the ProjectDisplayService - so they should both be in the same file.

Quickest way to fix this is probably to just copy the testDisplaySingle() method into this class, and then rename this test class to be ProjectDisplayServiceTest.

Generally it's just one test file per class